cmp/ToastManager.js

```diff
@@ -22,10 +22,10 @@ export const ToastManager = {
      * Show a 'Toast' message.
      *
      * @param {string} message - the message to show in the toast.
-     * @param {element} icon - icon to be displayed
-     * @param {number} timeout - time in milliseconds to display the message.
-     * @param {string} intent - the Blueprint intent.
-     * @param {string} position - position in viewport to display. See Blueprint Position enum. 
+     * @param {element} [icon] - icon to be displayed
```

Comment:
Is this `Element` as it's a class vs. a primitive? (Still not entirely sure of the exact nature of some of these things)

Reply:
Well, its a common, and very specific thing in the API, and it would be nice to have a specific name for it.  Strictly speaking its not an instance of class Element, but I think we would do well to refer to it as 'Element'.